portfolio/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I see why `make pretty` didn't work: this file must be at the root of your repo (not under `/portfolio`), and similarly, you need to run `make pretty` at the root of the repo as well.